EOSBasic/Plugins/UEOS/Source/UEOS/Public/Friends/Friends.h

```diff
@@ -20,8 +20,10 @@ enum class EFriendStatus : uint8
 	Friends = 3,
 };
 
-
 DECLARE_DYNAMIC_MULTICAST_DELEGATE( FOnFriendsRefreshed );
+DECLARE_DYNAMIC_MULTICAST_DELEGATE(FOnFriendInviteSent);
```

Comment:
Consider creating a "reason" enum, then having one Friends Delegate that passes the value. Fewer binds.

Reply:
Yes, good idea.